### PR TITLE
[RFC] Update terminal documentation for clarity

### PR DESCRIPTION
**Pull request by srobbibaro**
A couple lines tripped me up while reading through this document for the first
time. This change aims to reword/rework these areas, so that they are clearer
on the first read.

### CODE REVIEW

File: runtime/doc/nvim_terminal_emulator.txt [hunk 1]
```diff
@@ -62,8 +62,8 @@ with the terminal. For example, here's how to map <Esc> to exit terminal mode:
     :tnoremap <Esc> <C-\><C-n>
 <
 Navigating to other windows is only possible by exiting to normal mode, which
-can be cumbersome with <C-\><C-n> keys. Here are some mappings to improve
-the navigation experience:
+can be cumbersome with <C-\><C-n> keys. To improve the navigation experience,
```
Comment (Pyrohh) [inline runtime/doc/nvim_terminal_emulator.txt]: :+1:

**Comment by Pyrohh** [comment]
LGTM, I'll merge in a day if no one objects. While the benefit to clarity is minor, it's definitely there IMO.

**Comment by justinmk** [comment]
No objection, though in general we should try to aim for fewer, not more, words. The passive voice is also pretty difficult to read in general, and it is still present after this changes.

File: runtime/doc/nvim_terminal_emulator.txt [hunk 2]
```diff
@@ -62,8 +62,8 @@ with the terminal. For example, here's how to map <Esc> to exit terminal mode:
...
+the following mappings can be used:
```
Comment (fmoralesc) [inline runtime/doc/nvim_terminal_emulator.txt]: -> "you could use the following mappings:" ?

Comment (srobbibaro) [inline runtime/doc/nvim_terminal_emulator.txt]: I like that better - let me rebase the change in.

**Pull request merged**